tags/faq/launchers.ytag

```diff
@@ -22,6 +22,7 @@ embed:
         * [GDLauncher](https://gdlauncher.com/)
         * [ATLauncher](https://atlauncher.com/)
         * [CurseForge App](https://download.curseforge.com/)
+        * [Modrinth App](https://modrinth.com/app)
```

Comment:
The Modrinth App is still in beta.

Reply:
> The Modrinth App is still in beta.

only the windows version is listed as in beta, and even that is hidden unless a user scrolls. From the outside appearance, it appears that it is out of beta except for the single windows beta link if you scroll to it.

Reply:
Yes, the "beta" label is hidden. It's still in beta though, and not just the Windows version. 

I believe it was Propspector that said that it's the 0.x that denotes that it's in beta. Someone said something about it being obvious when someone pointed out that there's no in-app "beta" label

You can find all sorts of evidence in their Discord so I'll screenshot some:
<details><summary>Details</summary>

![Screenshot_20241129_094512](https://github.com/user-attachments/assets/48dee54c-53fd-4033-8b0b-c0a39779bb66)

</details>

<details><summary>Details</summary>

![Screenshot_20241129_095429](https://github.com/user-attachments/assets/9d21040b-42c3-4069-9a2c-9ef506991a40)

</details>

<details>

![Screenshot_20241129_095717](https://github.com/user-attachments/assets/dd3470de-0ed1-4b4a-a78d-e109a05fe44b)

</details>

https://github.com/modrinth/knossos/issues/1480

Reply:
hiding beta status from the average user means it is effectively not beta. if the only way to tell its in beta, for the average player, is to dig into it, join a discord, read a source code version control system's issue tracker, etc, then its not actually in beta. thats just an excuse when stuff fails.

If its in beta, it would be marked clearly on the website. 

If the concern is that a buggy production released launcher is being called beta only when people have problems, then that is a valid concern.